<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2020-08-24T20:39:33 EDT" name="Default suite" finished-at="2020-08-24T20:39:51 EDT" duration-ms="18512">
    <groups>
    </groups>
    <test started-at="2020-08-24T20:39:33 EDT" name="Default test" finished-at="2020-08-24T20:39:51 EDT" duration-ms="18512">
      <class name="tests.homepagetest">
        <test-method is-config="true" signature="reportSetup()[pri:0, instance:tests.homepagetest@22555ebf]" started-at="2020-08-24T20:39:32 EDT" name="reportSetup" finished-at="2020-08-24T20:39:33 EDT" duration-ms="298" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- reportSetup -->
        <test-method is-config="true" signature="setup()[pri:0, instance:tests.homepagetest@22555ebf]" started-at="2020-08-24T20:39:33 EDT" name="setup" finished-at="2020-08-24T20:39:40 EDT" duration-ms="6887" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="homepageTests()[pri:0, instance:tests.homepagetest@22555ebf]" started-at="2020-08-24T20:39:40 EDT" name="homepageTests" finished-at="2020-08-24T20:39:48 EDT" duration-ms="8703" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [30] but found [40]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [30] but found [40]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at org.testng.Assert.assertEquals(Assert.java:917)
at tests.homepagetest.homepageTests(homepagetest.java:28)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- homepageTests -->
        <test-method is-config="true" signature="getResult(org.testng.ITestResult)[pri:0, instance:tests.homepagetest@22555ebf]" started-at="2020-08-24T20:39:48 EDT" name="getResult" finished-at="2020-08-24T20:39:50 EDT" duration-ms="1579" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=homepageTests status=FAILURE method=homepagetest.homepageTests()[pri:0, instance:tests.homepagetest@22555ebf] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method is-config="true" signature="cleanup()[pri:0, instance:tests.homepagetest@22555ebf]" started-at="2020-08-24T20:39:50 EDT" name="cleanup" finished-at="2020-08-24T20:39:51 EDT" duration-ms="1287" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanup -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.homepagetest@22555ebf]" started-at="2020-08-24T20:39:51 EDT" name="tearDown" finished-at="2020-08-24T20:39:52 EDT" duration-ms="814" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- tests.homepagetest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
